core/src/components/label/label.md.scss

```diff
@@ -175,9 +175,9 @@
 ::slotted(*) p {
   @include margin(0, 0, 2px);
 
-  font-size: 14px;
+  font-size: dynamic-font(14px);
 
-  line-height: 20px;
+  line-height: dynamic-font(20px);
```

Comment:
Should this still use the Sass function? It does read a bit weird calling dynamic-_font_.